features/src/rapids-build-utils/opt/rapids-build-utils/bin/tmpl/python.build.wheel.tmpl.sh

```diff
@@ -82,6 +82,11 @@ build_${PY_LIB}_python_wheel() {
         pip_args+=(-C "build-dir=$(rapids-maybe-clean-build-dir "${cmake_args[@]}" -- "${PY_SRC}")");
     fi
 
+    # devcontainers builds always prefer to build projects unsuffixed, e.g. 'cudf' instead of 'cudf-cu12'
+    if rapids-python-uses-rapids-build-backend "${PY_SRC}"; then
+        pip_args+=(-C "rapidsai.disable-cuda=true");
+    fi
+
```

Comment:
Is this something we could encode in the `manifest.yaml` for the projects?

Reply:
Though now that I say that, I am recalling that we can encode arbitrary shell scripts from entries in `manifest.yaml` as well, since they get inlined and executed in the generated script.

This makes me want to move the `scikit-build-core` parts into `manifest.yaml` as well.

Reply:
oooooo interesting. I like the idea of not introducing another script for this and driving it from `manifest.yaml`.

Is `python.[].args.wheel` in `manifest.yaml` here supported already for that?

Like I see over in DLFW's manifest:

```yaml
python:
      - name: wholegraph
        sub_dir: python/pylibwholegraph
        args:
          wheel: *rapids_python_wheel_args_run
```

Reply:
Yeah exactly. There's also `args: { install: [...] }`, so we should be able to define it there

Reply:
oh awesome! Ok I'll try that, thank you.

Reply:
Done